core_backend/app/users/schemas.py

```diff
@@ -20,3 +20,14 @@ class UserCreateWithPassword(UserCreate):
     password: str
 
     model_config = ConfigDict(from_attributes=True)
+
+
+class UserRetrieve(BaseModel):
```

Comment:
Can you just extend UserCreate? Also, is `user_id` important to share back? I think it should be an internal thing and `username` should be the identifier that people get

Reply:
Hmm, maybe it should just be UserCreate... You're right, we don't need to return the user_id I don't think